src/scripts/browser.js

```diff
@@ -2,6 +2,11 @@ function isTv() {
     // This is going to be really difficult to get right
     const userAgent = navigator.userAgent.toLowerCase();
 
+    //The OculusBrowsers userAgent also has the samsungbrowser defined but is not a tv.
+    if(userAgent.indexOf('oculusbrowser') !== -1) {
```

Comment:
Please amend the commit with these changes and I think it's fine.
```suggestion
    // The OculusBrowsers userAgent also has the samsungbrowser defined but is not a tv.
    if (userAgent.indexOf('oculusbrowser') !== -1) {
```

Reply:
Sure, I have updated the commit.